api/src/functionalTest/java/uk/gov/hmcts/payment/functional/fixture/PaymentFixture.java

```diff
@@ -15,6 +15,9 @@
 
 public class PaymentFixture {
 
+    private static final int CCD_CASE_NUMBER_MIN_VALUE = 100000000;
+    private static final int CCD_CASE_NUMBER_MAX_VALUE = 999999999;
+
     public static CardPaymentRequest aCardPaymentRequest(String amountString) {
         String ccdCaseNumber = "1111-CC12-" + RandomUtils.nextInt();
```

Comment:
Why is this Data not changed and for the subsequent method as well.

Reply:
Only test cases for PBA payment API have been updated.